File: back/src/pro-atividade-api/Models/Atividade.cs
namespace pro_atividade_api.Models
{
    public class Atividade
    {
        public int id { get; set; }
        public string Titulo { get; set; }
        public string Descricao { get; set; }
        public string Prioridade { get; set; }

        public Atividade()
        {

        }
    }
}

File: front/pro-atividade-app/src/model/atividade.ts
export interface atividade{
    id: number;
    prioridade: string;
    titulo: string;
    descricao: string;
  }

export interface AtividadeListaProps{
    atividades: atividade[];
    pegarAtividade: (id: number) => void;
    handleConfirmModal: (id: number) => void;
  }
  